docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -356,26 +292,91 @@ export default async function buildInterfacesDocumentation(
       // eslint-disable-next-line no-console
       console.log('Built JSON file for', parsedInterface.name);
     } else {
+      const content = {
+        name: parsedInterface.name,
+        imports: generateImportStatement(parsedInterface, projects),
+        ...extractDemos(parsedInterface.tags.demos),
+        properties: {},
+      };
+
+      const translations = {
+        interfaceDescription: renderMarkdown(
+          escapeCell(linkify(parsedInterface.description, documentedInterfaces, 'html')),
+        ),
+        propertiesDescriptions: {},
+      };
+
+      parsedInterface.properties
+        .map((property) => ({
+          name: property.name,
+          description: renderMarkdown(
+            escapeCell(linkify(property.description, documentedInterfaces, 'html')),
+          ),
```

Comment:
```suggestion
          description: renderMarkdown(linkify(property.description, documentedInterfaces, 'html'),
          ),
```

`escapeCell` breaks the links:

<img width="884" src="https://github.com/mui/mui-x/assets/13808724/05ff8bda-3d30-4060-a9e7-f9b9c8b84063">

https://deploy-preview-12016--material-ui-x.netlify.app/x/api/data-grid/grid-api/#grid-api-prop-upsertFilterItems

Reply:
I ended up reordering the `escapeCell` and `linkify` functions, so that the links are only added after the string is escaped.